arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -147,6 +155,10 @@ void BenchFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                      "filename to write junit style report to",
                      new StringParameter(&_junitReportFile));
 
+  options->addOption("--json-report-file",
+                     "filename to write a json report to",
```

Comment:
Make it less of a tautology?
```suggestion
                     "filename to write a report in JSON format to",
```
Should the JSON structure be described in the docs? It's probably self-descriptive.

Reply:
hm, at the time being arangobench is a development tool; don't know whether it needs this.

Reply:
Yeah, should be fine.